impeller/aiks/picture.cc

```diff
@@ -76,7 +77,8 @@ std::shared_ptr<Texture> Picture::RenderToTexture(
         /*mip_count=*/1,
         "Picture Snapshot",                           // label
         RenderTarget::kDefaultColorAttachmentConfig,  // color_attachment_config
-        std::nullopt  // stencil_attachment_config
+        RenderTarget::
+            kDefaultStencilAttachmentConfig  // stencil_attachment_config
```

Comment:
Just these changes in `picture.cc` are needed to get us over the line.

Reply:
The remaining backdrop blur goldens for GLES were fixed with https://github.com/flutter/engine/pull/51173